packages/nova-graphql-compiler/README.md

```diff
@@ -19,3 +19,5 @@ Then inside your package.json add the compiler to your `scripts`:
   }
 }
 ```
+
```

Comment:
This sounds like something we should fix?

Reply:
I agree but who has funding to fix it? I created an issue a while back in https://github.com/microsoft/graphitation/issues/331